### DISCUSSION TITLE
how to build a VUE project

### DISCUSSION THREAD

**Discussion by Sunny-001** [paketo-buildpacks/feedback#46] [General]
dear team，I use the builder "parketobuildpacks/builder:base" to build a vue project, and then get an image. When I run the image, the following error occurs. I don't know if I'm using the wrong method. I need some help. Thank you very much.

Some contents of package.json are as follows:
![image](https://user-images.githubusercontent.com/46052043/200760027-5ffce0b2-b35d-4ed4-a107-414fb67c4ebb.png)

when I run "docker run - it my image", I get the following error:
![image](https://user-images.githubusercontent.com/46052043/200762266-b53ddf60-f29d-4ff2-9323-40043acd2279.png)

**Comment by dmikusa** [comment]
Take a look at https://github.com/paketo-buildpacks/samples/tree/main/nodejs/vue-npm

**Reply by Sunny-001** [reply]
thank you, It's helpful. i have another question.
I used the following command to build the vue project, and when I built it on another server with the same command, it still took longer than not using cache-image

command:
pack build registry.cn-hangzhou.aliyuncs.com/kpack/ruoyi-ui:2.0.0
--path ./ruoyi-ui/
--builder registry.cn-hangzhou.aliyuncs.com/kpack/paketo-buildpacks-builder:base
--env "BP_NODE_RUN_SCRIPTS=build"
--env "NODE_ENV=development"
--pull-policy if-not-present
--cache-image registry.cn-hangzhou.aliyuncs.com/kpack/nodejs-cache:latest
--publish

**Reply by dmikusa** [reply]
I responded on your other discussion thread.
